ui/app/clinical/consultation/controllers/dispositionController.js

```diff
@@ -32,9 +32,11 @@ angular.module('bahmni.clinical')
             });
         };
 
-        var findAction = function (dispositions, action) {
-            var undoDischarge = _.find(dispositions, action);
-            return undoDischarge || {'name': ''};
+        var pushDispositionAction = function (finalDispositionActions, dispositions, action) {
+            var dispositionPresent = _.find(dispositions, action);
+            if (dispositionPresent) {
+                finalDispositionActions.push(dispositionPresent);
```

Comment:
Hi @maharjunm,
**pushDispositionAction** method should not modify the parameter (**finalDispositionActions**). It should return some value